backend/prompt_studio/prompt_studio_core/urls.py

```diff
@@ -77,7 +79,9 @@
         ),
         path(
             "prompt-studio/index-document/<uuid:pk>",
-            prompt_studio_prompt_index,
+            method_decorator(transaction.non_atomic_requests)(
+                prompt_studio_prompt_index
+            ),
```

Comment:
@johnyrahul is this to make this API transaction as non atomic? If yes please confirm with @chandrasekharan-zipstack if it would be safe to do so.

Reply:
yes @ritwik-g . @chandrasekharan-zipstack  please check and let me know if this is ok?

Reply:
It seems like this has to be applied on the django View - this seems to be valid